packages/xrpl/HISTORY.md

```diff
@@ -4,6 +4,9 @@ Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xr
 
 ## Unreleased
 
+### Fixed
+* Update packages to resolve potential security issues.
```

Comment:
Is this line required in History change-log?  Do the changes in this PR affect external APIs or end-users?

If you'd still like to retain this line in the HISTORY file, I'd prefer to see the new dependency `name@version` information. That is helpful for future debugging purposes.

Reply:
Oh you're right. We can remove this line.